### ISSUE TITLE
create did not work for me

### ISSUE THREAD

**Issue by Farson** [Toptranslation/hermes#2]
I tried: 

```
./hermes.py -v create -p meike.gressel --mail meike.gressel@toptranslation.com --password 'some_password' --department "Account Management" --title "Account Manager"
```

didn't work :(

**Comment by Farson** [comment]
```
wg@tux:~/hermes (master)$ ./hermes.py -v create -p meike.gressel --mail meike.gressel@toptranslation.com --password 'geheim' --department "Account Management" --title "Account Manager"
Namespace(COMMAND='create', UID='meike.gressel', base='dc=toptranslation,dc=com', bots=False, botsdn='ou=bots', clients=False, clientsdn='ou=clients', department='Account Management', description=None, environment='DEFAULT', formerdn='ou=former workers', groupsdn='ou=internal groups', localport=3890, mail=['meike.gressel@toptranslation.com'], mobile=None, password='tkd4hjx8iu29', people=True, peopledn='ou=people', port=389, server='85.214.231.127', sshport=40922, ssl=False, telephone=None, title='Account Manager', username='waldemar.gribele', verbose=True)
[INFO] Connecting to SSH...
[INFO] Forwarding LDAP Port
Password for the user uid=waldemar.gribele,ou=people,dc=toptranslation,dc=com: 
[INFO] Connected! Tunnel open ('127.0.0.1', 52478) -> ('85.214.231.127', 40922) -> ('127.0.0.1', 389)
[INFO] LDAP add:
    uid=meike.gressel,ou=people,dc=toptranslation,dc=com
    object_class = inetOrgPerson
    attributes = {'title': 'Account Manager', 'sn': 'Gressel', 'initials': 'mg', 'givenName': 'Meike', 'mail': ['meike.gressel@toptranslation.com', 'meike.gressel@toptranslation.com'], 'uid': 'meike.gressel', 'cn': 'Meike Gressel', 'ou': 'Account Management', 'userPassword': '{SSHA}8hpJaiagUi1aw2K+bnD7187h+8SiVMpZ', 'displayName': 'Meike Gressel'}
[INFO] {'type': 'addResponse', 'result': 20, 'dn': '', 'referrals': None, 'message': 'mail: value #0 provided more than once', 'description': 'attributeOrValueExists'}
[INFO] LDAP modify_add:
    cn=owncloud,ou=internal groups,dc=toptranslation,dc=com
    member: uid=meike.gressel,ou=people,dc=toptranslation,dc=com
[INFO] {'type': 'modifyResponse', 'result': 20, 'dn': '', 'referrals': None, 'message': 'modify/add: member: value #0 already exists', 'description': 'attributeOrValueExists'}
[INFO] LDAP modify_add:
    cn=redmine,ou=internal groups,dc=toptranslation,dc=com
    member: uid=meike.gressel,ou=people,dc=toptranslation,dc=com
[INFO] {'type': 'modifyResponse', 'result': 20, 'dn': '', 'referrals': None, 'message': 'modify/add: member: value #0 already exists', 'description': 'attributeOrValueExists'}
[INFO] Tunnel closed from ('127.0.0.1', 52478)
wg@tux:~/hermes (master)$ 
```

**Comment by tequir00t** [comment]
I will update the documentation for the next commit, and will show an error if the same email is included twice.
